Documentation/QuickInstall/Composer/Index.rst

```diff
@@ -5,44 +5,85 @@
 
 .. attention::
 
-   The recommended way of installing TYPO3 is via Composer, as described on
-   this page.
+   The recommended way of installing TYPO3 is via Composer as described on this
+   page.
 
 
-To create a new TYPO3 project use the TYPO3 Base Distribution::
+Using Composer
+==============
 
-	# Download the Base Distribution, the latest "sprint" release (10)
-	composer create-project typo3/cms-base-distribution YourNewProjectFolder
+*NIX Environment
+----------------
+
+In *NIX shells the Asterisk character ``*`` has a special meaning and has to be
+properly escaped by surrounding double quotes ``"``:: shell
+
+   composer create-project vendor/package YourNewProjectFolder "*"
+   composer create-project "vendor/package:*" YourNewProjectFolder
+
+
+Windows Environment
+-------------------
+
+To install TYPO3 via the ``composer`` command on Windows, it should be started
+as admin or explicitly given the right to create symlinks (use for example a
+powershell or git bash started with admin rights).
+
+The Caret character ``^`` is the escape character in a Windows command line
+(cmd.exe). To be able to use it in version constraints, every occurance has to
+be escaped by prepend a Caret character ``^`` or by enclosing the whole
+argument in double quotes ``"``:: shell
+
+   # Escaping by prepend the escape character
+   composer create-project typo3/cms-base-distribution:^^9 YourNewProjectFolder
+
+   # Escaping by enclosing in double quotes
+   composer create-project "typo3/cms-base-distribution:^9" YourNewProjectFolder
 
-.. note::
 
-   To install TYPO3 via the `composer` command on Windows, it should be started as admin
-   or explicitly given the right to create symlinks (use for example a
-   powershell or git bash started with admin rights).
+Install a TYPO3 LTS Release
+===========================
 
-.. important::
+To create a new TYPO3 project use the TYPO3 Base Distribution including a
+version constraint::
 
-   The caret character `^` is the escape character in a Windows command line (cmd.exe). To
-   be able to use it in version constraints, every occurance has to be escaped by adding a
-   second caret `^` or by enclosing the whole argument in double quotes `"`:
+	# Download the Base Distribution using TYPO3 LTS release version 9.5.x
+	composer create-project typo3/cms-base-distribution:^9 YourNewProjectFolder
 
-   * composer create-project typo3/cms-base-distribution:^^9 YourNewProjectFolder
-   * composer create-project "typo3/cms-base-distribution:^9" YourNewProjectFolder
+	# Download the Base Distribution using TYPO3 LTS release version 8.7.x
+	composer create-project typo3/cms-base-distribution:^8 YourNewProjectFolder
+
+All available versions are listed on `Packagist
+<https://packagist.org/packages/typo3/cms-base-distribution>`__. Using the
+MAJOR version, the first number of the version see `Semantic Versioning
+<https://semver.org>`__, normally is sufficient.
+
+
+Install a TYPO3 Sprint Release
```

Comment:
I would name this maybe "Install **latest** TYPO3 Release" to make this more clear. Once 10 LTS is out, installing without version constraint would install the latest LTS so I find the phrasing of the titles a little misleading currently.

Reply:
Good point, so the note at the end can be simplified too

Reply:
Question: Do you have plans to mention the possiblity to prepare a `composer.json` that includes all the packages needed? (Some extensions bring examples of such setups, e.g. aimeos, and also the ddev video tutorials presents such a `composer.json`, but in v8 [with the `extra` key].) IMO it should be mentioned and it should be pointed out clearly that there are important differences between v8 and v9. Also, it should be explained that only `typo3/minimal` is a valid preset, but `typo3/full` and `typo3/default` are (currently) not (see this page, which could easily create confusion: https://get.typo3.org/misc/composer/helper).

Reply:
@rowild I wasn't even aware of https://get.typo3.org/misc/composer/helper. I also don't know about typo3/full and typo3/default. I think it might be a good idea to talk with the people who do get.typo3.org and also composer.typo3.org. Some things are in fact duplicate and contradict each other. I would maybe create a separate issue / PR for this (but on second thought there is already an issue, which could be used: https://github.com/TYPO3-Documentation/T3DocTeam/issues/84) The topic composer expands across several manuals. I  expanded it. Feel free to comment there. If you can, please point out the differences between v8 and v9.